ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh

```diff
@@ -52,9 +52,20 @@ fi
 
 # summarize test results
 echo "Summarizing test results..."
-[[ -e "${ARTIFACT_DIR}" ]] || exit 0
+if ! [[ -d "${ARTIFACT_DIR:-'/default-non-exist-dir'}" ]] ; then
+    echo "Artifact dir '${ARTIFACT_DIR}' not exist"
+    exit 0
+else
+    echo "Artifact dir '${ARTIFACT_DIR}' exist"
+    ls -lR "${ARTIFACT_DIR}"
+    files="$(find "${ARTIFACT_DIR}" -name '*.xml' | wc -l)"
+    if [[ "$files" -eq 0 ]] ; then
+        echo "There are no JUnit files"
+        exit 0
+    fi
+fi
 declare -A results=([failures]='0' [errors]='0' [skipped]='0' [tests]='0')
-grep -r -E -h -o 'testsuite.*tests="[0-9]+"[^>]+' "${ARTIFACT_DIR}/gui_test_screenshots/console-cypress.xml" 2>/dev/null > /tmp/zzz-tmp.log
+grep -r -E -h -o 'testsuite.*tests="[0-9]+"[^>]*' "${ARTIFACT_DIR}/gui_test_screenshots/console-cypress.xml" 2>/dev/null > /tmp/zzz-tmp.log || exit 0
```

Comment:
Change from `[^>]+` to `[^>]*` because `tests="123"` can exist at the end of the line.
Also add `exit 0` so that if the grep command can not find the pattern ( then we don't know how to parse the result), exit successfully, so that the other tests following this step can move on.